src/styles/blocks/container_position.scss

```diff
@@ -0,0 +1,9 @@
+.catalog_container {
```

Comment:
`catalog` - BEM block
`catalog_container` - element of of catalog

File should have the same name as block so rename to `catalog`